rct229/rulesets/ashrae9012019/ruleset_functions/get_baseline_system_types.py

```diff
@@ -99,33 +99,30 @@ def get_baseline_system_types(rmi_b):
     )
     dict_with_terminal_units_and_zones = get_dict_with_terminal_units_and_zones(rmi_b)
 
-    for hvac_b in find_all("$..heating_ventilating_air_conditioning_systems[*]", rmi_b):
+    for hvac_b in find_all(
+        "$.buildings[*].building_segments[*].heating_ventilating_air_conditioning_systems[*]",
+        rmi_b,
+    ):
         hvac_b_id = hvac_b["id"]
         terminal_unit_id_list = dict_of_zones_and_terminal_units_served_by_hvac_sys[
             hvac_b_id
         ]["terminal_unit_list"]
         zone_id_list = dict_of_zones_and_terminal_units_served_by_hvac_sys[hvac_b_id][
             "zone_list"
         ]
-        # one terminal per zone for all baseline systems
```

Comment:
This `if` statement is always true because each terminal _lives_ inside a single zone.  So I was able to simplify the logic.  The test file was also simplified since there is one fewer branch to test.